build.fsx

```diff
@@ -93,4 +107,15 @@ pipeline "Verify" {
     runIfOnlySpecified true
 }
 
+pipeline "InstallTool" {
```

Comment:
I don't quite get why you need this. What exactly are you trying to achieve here?

Reply:
It's not strictly needed. But, my goal was to make it easy to use the same pipeline that is used to test the tool that results from `dotnet pack` so that testing locally and what happens in CI are aligned.

I can revert the `build.fsx` changes if invoking the executable is the recommended way to test locally though.

Reply:
Yes, please revert this. Running the binary locally to test should suffice.

Reply:
Done!